www/_posts/2021-08-21-removal-of-translated-docs.md

```diff
@@ -0,0 +1,22 @@
+---
+layout: post
+author:
+    name: Bryan Ellis
+title:  "Notice of the Removal of the Translated Docs"
+categories: announcements
+tags: news
+---
+
+We like to announce that we have deprecated and removed all translated docs.
+
+We had been providing translated docs since Cordova CLI 3.1.0 but, the number of resources has decreased to a point where it affects our ability to maintain them. The maintenance of these translated docs started to diverge around Cordova CLI 6.x.
+
+We have updated our webserver to redirect the language docs to their English counterparts. A 404 page will display only when the English counterparts do not exist. This issue occurs when translated docs continue to show deprecated content. For example, the BlackBerry 10 platform was deprecated and removed from Cordova CLI 8.x but, the translations were not updated and still show the deprecated platform.
+
+We like to thank the people who took the initiative in creating the translated docs and those who had put in their time and effort into maintaining them.
```

Comment:
"We like" -> "We'd like"

Reply:
I changed it to We'd as suggested but, I am making a note that we are removing. We are not talking about would like to remove. So the tone of this is wrong and lying.

Reply:
I changed it to`We'd` as suggested.

Reply:
Thanks!